slack_bot/tests/token_rotation_test.py

```diff
@@ -0,0 +1,75 @@
+# """
+# Unit tests for the app config token rotation logic
+# """
+# import unittest
+# from pydantic import SecretStr
+# from pydantic_settings import BaseSettings
+# from dotenv import load_dotenv, find_dotenv, dotenv_values
+#
+# from slack_bot.token_rotation import rotate_token
+#
+# dotenv_path = find_dotenv()
+# load_dotenv(dotenv_path)
+# env_values = dotenv_values(dotenv_path)
+#
+#
+# class Settings(BaseSettings):
+#     """Environment variables settings"""
+#     access_token: SecretStr
+#     signing_secret: SecretStr
+#     config_data_dir: str = './config_data/'
+#     app_config_token: SecretStr = SecretStr('')
+#     bot_app_id: str = ''
+#     refresh_token: SecretStr = SecretStr('')
+#
+#     class Config:
+#         env_file = '.env'
+#         env_prefix = 'slack_'
+#         case_sensitive = False
+#
+#
+# class TestRotateToken(unittest.TestCase):
+#
+#     def setUp(self) -> None:
```

Comment:
why is it commented?

Reply:
Please create a separate issue for that and remove commented code it from this PR

Reply:
Done.